### PR TITLE
MGDOBR-1199 Run Performance tests on dev cluster

### PR DESCRIPTION
**Pull request by sutaakar**
Signed-off-by: Karel Suta <ksuta@redhat.com>

There are still some Horreum leftovers, I will remove them once a complete CI to upload to Horreum will be implemented.

[MGDOBR-1199](https://issues.redhat.com/browse/MGDOBR-1199)

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] The layers in the `kustomize` folder have been updated accordingly.
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

File: integration-tests/manager-integration-tests/README.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ You can update the parameters needs to configure in `integration-tests/manager-i
 
 In order to be able to run the [performance tests scenarios](src/test/resources/performance-features) locally you will need to have the following installed on your machine:
 
-- [Hyperfoil v0.21](https://hyperfoil.io/) - Benchmark framework for microservices (*)
+- Latest [Hyperfoil](https://hyperfoil.io/) - Benchmark framework for microservices (*)
```
Comment (jiripetrlik) [inline integration-tests/manager-integration-tests/README.md:123]: Is it stable to always use the latest version. Isn't using the fixed version more consistent and stable?

File: integration-tests/manager-integration-tests/src/test/resources/performance-features/continuously-create-bridges.feature [hunk 2]
```diff
@@ -134,6 +136,8 @@ Feature: Continuous create Bridges performance tests
     When run benchmark with content:
       """text/vnd.yaml
       name: rhose-continuously-create-bridges-with-error-handler
+      agents:
+        driver01:
```
Comment (jiripetrlik) [inline integration-tests/manager-integration-tests/src/test/resources/performance-features/continuously-create-bridges.feature:140]: Is the missing value expected?

**Review by jiripetrlik — COMMENTED** [review]
Looks good to me. I have some minor comments/questions.

File: integration-tests/manager-integration-tests/README.md (hunk 1 above)
Comment (sutaakar) [reply integration-tests/manager-integration-tests/README.md:123]: hmm, good point, will mention specific version

**Review by jiripetrlik — APPROVED** [review]

**Review by ChughShilpa — APPROVED** [review]

**Pull request merged**